src/controllers/searchCompany.controller.ts

```diff
@@ -39,12 +41,13 @@ export class SearchCompanyController extends BaseController {
 
   @httpPost('')
   public async post(@requestBody() body: SearchCompanyFormModel): Promise<string | RedirectResult> {
+    body.companyNumber = this.companyNumberSanitizer.sanitizeCompany(body.companyNumber!)
     const errors: Optional<ValidationErrors> = this.validator.validate(body, formSchema)
```

Comment:
I believe the validation schema has some length checks in here. can you remove any redundant checks if any exist?